rclcpp_action/README.md

```diff
@@ -0,0 +1,7 @@
+# `rclcpp_action`
+
+Adds action APIs for C++.
```

Comment:
Add link to action documentation:

http://design.ros2.org/articles/actions.html